pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml

```diff
@@ -1462,7 +1462,7 @@ spec:
                         type: string
                       type: array
                     fromCIDRGroupRefs:
-                      description: FromCIDRGroupRefs is a list of CIDRs which the
+                      description: FromCIDRGroupRef is a list of CIDRs which the
```

Comment:
Nit: What's the reason to rename it to singular form? I think it was accurate as plural because the user can insert multiple refs in the same `fromCIDRGroupsRefs` field.

Reply:
That was my initial thought, but looking at other similar CNP fields (e.g.: `fromCIDR` and `fromCIDRSet`) that are reported as singular despite being slices, I thought to change `FromCIDRGroupRef` too.
I left it as a separate commit for now, if you think that the plural version is better, I can revert it easily.